src/Layouts/MainView/Main.tsx

```diff
@@ -1,15 +1,12 @@
-import { FC } from "react";
-import { Navbar } from "../../components/Navbar/Navbar";
-import { ReactChildrenProps } from "../../types/types";
-import "./Main.scss";
-
-
+import { FC } from 'react';
+import { Wrapper, Navbar } from '../../components';
+import { ReactChildrenProps } from '../../types/types';
+import './Main.scss';
 
 export const Main: FC<ReactChildrenProps> = ({ children }) => {
   return (
-    <div className="mainWrapper">
-      <div className="childrenBlock">{children}</div>
-      <Navbar />
+    <div className='mainWrapper'>
```

Comment:
For consistency can we use one way of styling? If we use styled components it would be better to stick with it

Reply:
Discussed